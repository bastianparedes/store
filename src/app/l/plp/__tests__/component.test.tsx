import React from 'react';

import { render } from '@testing-library/react';

import Component from '../component';

const products = [
  {
    description: 'description1',
    name: 'name1',
    ownerEmail: 'test1@test.cl',
    price: 1000,
    quantity: 10,
    sku: 1
  },
  {
    description: 'description2',
    name: 'name2',
    ownerEmail: 'test2@test.cl',
    price: 2000,
    quantity: 20,
    sku: 2
  }
];

it('should render', () => {
  const { container } = render(<Component products={products} />);
  expect(container).toMatchSnapshot();
});
