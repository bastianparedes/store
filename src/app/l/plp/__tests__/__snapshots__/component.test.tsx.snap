// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Cat√°logo
    </h1>
    <div
      class="productList"
    >
      <div
        class="productItem"
      >
        <a
          href="/l/pdp/1"
        >
          <img
            alt="name1"
            src="/api/image/1"
          />
        </a>
        <p>
          Nombre: 
          name1
        </p>
        <p>
          Precio: $
          1000
        </p>
        <p>
          Cantidad disponible: 
          10
        </p>
      </div>
      <div
        class="productItem"
      >
        <a
          href="/l/pdp/2"
        >
          <img
            alt="name2"
            src="/api/image/2"
          />
        </a>
        <p>
          Nombre: 
          name2
        </p>
        <p>
          Precio: $
          2000
        </p>
        <p>
          Cantidad disponible: 
          20
        </p>
      </div>
    </div>
  </div>
</div>
`;
